sdk/src/main/java/com/sygic/travel/sdk/Sdk.kt

```diff
@@ -11,7 +11,7 @@ import com.sygic.travel.sdk.tours.facade.ToursFacade
 /**
  * Provides public methods for requesting API.
  */
-class StSDK(xApiKey: String, context: Context) {
+class Sdk(xApiKey: String, context: Context) {
```

Comment:
please update also readme for this change :)

Reply:
:heavy_check_mark: